lib/ts/recipe/oauth2client/types.ts

```diff
@@ -78,24 +78,25 @@ export type TypeNormalisedInput = {
 };
 
 export type RecipeInterface = {
-    getAuthorisationRedirectURL(input: {
-        redirectURIOnProviderDashboard: string;
-    }): Promise<{
+    getAuthorisationRedirectURL(
+        redirectURIOnProviderDashboard: string,
+        userContext: UserContext
+    ): Promise<{
         urlWithQueryParams: string;
         pkceCodeVerifier?: string;
     }>;
-    getProviderConfig(input: { userContext: UserContext }): Promise<ProviderConfigWithOIDCInfo>;
+    getProviderConfig(userContext: UserContext): Promise<ProviderConfigWithOIDCInfo>;
 
-    signIn(input: {
-        userId: string;
-        oAuthTokens: OAuthTokens;
+    signIn(
+        userId: string,
+        oAuthTokens: OAuthTokens,
         rawUserInfoFromProvider: {
             fromIdTokenPayload?: { [key: string]: any };
             fromUserInfoAPI?: { [key: string]: any };
-        };
-        tenantId: string;
-        userContext: UserContext;
-    }): Promise<{
+        },
+        tenantId: string,
+        userContext: UserContext
+    ): Promise<{
```

Comment:
These should still be getting the input object.

Reply:
I have fixed it.